zebra-state/src/service/check/utxo.rs

```diff
@@ -63,6 +63,7 @@ pub fn transparent_spend(
                 finalized_state,
             )?;
 
+            // FIXME: what about v6?
```

Comment:
Remove comment and fix line `69` to be
`// This is a particular issue for v5 and v6 transactions,`